connected_users/frame.py

```diff
@@ -0,0 +1,57 @@
+import psutil as ps
+import tkinter as tk
+from integrator import frame
+from logger.logger import Logger
+from datetime import datetime
+
+
+class frame(frame.DOSMFrame):
```

Comment:
Suggestion par rapport à la proposition de @RedbeanGit : [la proposition en question](https://github.com/do3-2021/dosm/pull/20/files/3c1b6b1d9877045c20d9fff076ef7bbb5e7bfd68#r810614502)

```suggestion
class Tab(frame.DOSMFrame):
```